docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,215 @@
+## Deployment
+This document covers the steps on setting up this repository on various cloud hosting providers.
+
+  - [AWS](#aws)
+
+# AWS
+
+## Pre-requisites
+
+Deploying on AWS requires a basic understanding of the following tools and services:
+1. Docker
+2. GitHub actions & workflows
+3. GitHub environments and secrets
+4. AWS Elastic Container Registry (ECR)
+5. AWS Elastic Container Service (ECS)
+6. AWS Virtual Private Cloud (VPC)
+7. AWS Fargate
+8. AWS Elastic Load Balancer (ELB)
+9. AWS Elastic IPs
+10. AWS Identity and Access Management (IAM)
+11. AWS Relational Database Service (RDS)
+
+## Staging
+
+### Continuous Delivery process
+
+An overview of how the continuous delivery cycle works in Plio with GitHub action and Amazon ECR & ECS.
```

Comment:
Have we set it up for multi-tenancy?

Reply:
https://cube.dev/docs/multitenancy-setup#top

Reply:
Working on it.